tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -310,10 +317,18 @@ def fit(train_data,
             where `L` ranges from 0 to 50 (Note: higher values of `L` correspond to fewer print statements, opposite of verbosity levels)
 
         Kwargs can include additional arguments for advanced users:
+            AG_args : dict, default={}
+                Keyword arguments to pass to all models. See the `AG_args` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
+                Identical to specifying `AG_args` parameter for all models in `hyperparameters`.
+                If a key in `AG_args` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
             AG_args_fit : dict, default={}
                 Keyword arguments to pass to all models. See the `AG_args_fit` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
                 Identical to specifying `AG_args_fit` parameter for all models in `hyperparameters`.
                 If a key in `AG_args_fit` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+            AG_args_ensemble : dict, default={}
+                Keyword arguments to pass to all models. See the `AG_args_ensemble` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
+                Identical to specifying `AG_args_ensemble` parameter for all models in `hyperparameters`.
+                If a key in `AG_args_ensemble` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
```

Comment:
Unambiguous wording suggestion: same parameters passed in `hyperparameters` will override values in `AG_args_ensemble`.

Reply:
I believe the existing description is unambiguous, although it's hard to accurately describe it simply.

These arguments are specific to `AG_args_ensemble`, ex:

```
hyperparameters={'GBM': {'num_boost_round': 10, 'AG_args_ensemble': {'max_base_models': 5}}}
```

Combined with:

```
AG_args_ensemble={'max_base_models': 20, 'max_base_models_per_type': 10}
```

Will result in:

```
hyperparameters={'GBM': {'num_boost_round': 10, 'AG_args_ensemble': {'max_base_models': 5, 'max_base_models_per_type': 10}}}
```

Reply:
I think it's unambiguous enough